.github/workflows/cilium-integration-tests.yaml

```diff
@@ -144,6 +144,19 @@ jobs:
             exit 1
           fi
 
+      - name: Install Go
+        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+        with:
+          # renovate: datasource=golang-version depName=go
+          go-version: 1.21.0
+
+      - name: Get new proxy API
```

Comment:
This is to make sure Go API updates cause no regression on the cilium/cilium build. Sometimes it is unavoidable (like when a deprecated feature is removed), but usually we do not want API update to break the build.

We need to do this step in a corresponding cilium/cilium Envoy update PR anyway, but having it here provides an automated heads-up on any breaking changes. This may help avoid making them if unintentional.

Reply:
yeah now i get it - we're making this check in the cilium/cilium repo that we checked out.